### PR TITLE
Replace clazz.newInstance() with clazz.getDeclaredConstructor().newInstance()

### PR DESCRIPTION
**Pull request by ritonglue**
newInstance should not be used. It becomes deprecated since java 9

### CODE REVIEW

File: src/main/java/com/quirk/csv/processor/CSVNamedReadProcessor.java [hunk 1]
```diff
@@ -49,18 +49,18 @@ protected void process(Reader reader, CSVFormat format, Consumer<T> consumer) th
             while(iterator.hasNext()){
                 consumer.accept(getCSVRecord(iterator,map));
```
Comment (bpoole6) [inline src/main/java/com/quirk/csv/processor/CSVNamedReadProcessor.java:50]: There's a compilation error here

**Review by bpoole6 — COMMENTED** [review]
Not changing the target compilation to java 11.

File: src/main/java/com/quirk/csv/processor/CSVNamedReadProcessor.java (hunk 1 above)
Comment (ritonglue) [reply src/main/java/com/quirk/csv/processor/CSVNamedReadProcessor.java:50]: Curious, my branch compiles.

**Comment by ritonglue** [comment]
I didn't want to change the compilation target. I wanted to remove the call to newInstance.

**Pull request closed without merging**